addon/utils/utils.js

```diff
@@ -74,7 +72,8 @@ export function getValidatableValue(value) {
   }
 
   if (isDSManyArray(value)) {
-    return emberArray(value.filter((v) => isValidatable(v)));
+    value.content = value.content.filter((v) => isValidatable(v));
```

Comment:
`content` is private and not stable. the other option of `toArray` is also not valid. Just treat EmberData things as native arrays, unless its a PromiseManyArray in which case you should error. PromiseManyArray can be detected in 5.0+ by something being a thenable and not having usual array methods but having a forEach method.